spacy/lang/ko/__init__.py

```diff
@@ -42,7 +42,10 @@ def check_spaces(text, tokens):
 class KoreanTokenizer(DummyTokenizer):
     def __init__(self, cls, nlp=None):
         self.vocab = nlp.vocab if nlp is not None else cls.create_vocab(nlp)
-        self.Tokenizer = try_mecab_import()
+        self.MeCab = try_mecab_import()("-F%f[0],%f[7]")
```

Comment:
So what we're assigning here now is the tokenizer instance, right?

Can we write this as:

```python

MeCab = try_mecab_import()
self.mecab_tokenizer = MeCab("-F%f[0],%f[7]")
```

It seems a bit clearer, otherwise it looks like we're assigning the module or the class instead.

Reply:
I've added a new commit fixing this.